libs/unity/library/Runtime/Scripts/NativeRender/NativeVideo.cs

```diff
@@ -84,13 +96,26 @@ public void EnableRemoteVideo(VideoKind format, TextureDesc[] textures)
 
             NativeVideoInterop.EnableRemoteVideo(_nativeVideoHandle, format, interopTextures, interopTextures.Length);
         }
+        
+        public void UpdateTextures(TextureDesc[] textures)
+        {
+            var interopTextures = textures.Select(item => new NativeVideoInterop.TextureDesc
+            {
+                texture = item.texture,
+                width = item.width,
+                height = item.height
+            }).ToArray();
+
+            NativeVideoInterop.UpdateTextures(_nativeVideoHandle, interopTextures);
+        }
 
         /// <summary>
         /// Stops rendering the remote video stream.
         /// </summary>
         public void DisableRemoteVideo()
         {
             NativeVideoInterop.DisableRemoteVideo(_nativeVideoHandle);
+            if (lookupDictionary.ContainsKey(_nativeVideoHandle)) lookupDictionary.Remove(_nativeVideoHandle);
```

Comment:
No need to check for existing key, [`Remove()` will fail returning `false` if it doesn't exist](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.remove?view=netstandard-2.0).
```suggestion
            lookupDictionary.Remove(_nativeVideoHandle);
```